charts/knuto/values.yaml

```diff
@@ -7,12 +7,18 @@ kafkauser_source_namespaces:
   production:
     deletion_enabled: false
     allowed_non_namespaced_topics: []
+    cross_namespace_read_allowed: true
+    read_allowed_cross_namespace_topics: []
   latest:
     deletion_enabled: false
     allowed_non_namespaced_topics: []
+    cross_namespace_read_allowed: true
+    read_allowed_cross_namespace_topics: []
   dev:
     deletion_enabled: true
     allowed_non_namespaced_topics: []
+    cross_namespace_read_allowed: true
```

Comment:
Should default to false?

Reply:
I think it is better to make that kind of configuration when using the chart, as the convention in KNUTO is to allow read from all namespaces:

"On the permissions/ACL side, it is assumed that a KafkaUser in any namespace is allowed to read data in all topics in the Kafka cluster, but only a KafkaUser in the namespace associated with a topic (i.e where the topic name is prefixed with the namespace name) is allowed to write to a topic. "

Reply:
Ok. Lgtm!